llm_demo/orchestrator/langchain_tools/langchain_tools_orchestrator.py

```diff
@@ -31,7 +31,12 @@
 from pytz import timezone
 
 from ..orchestrator import BaseOrchestrator, classproperty
-from .tools import initialize_tools
+from .tools import (
+    BASE_URL,
```

Comment:
Will probably not need these imports if we move the insert_ticket into tools.py.

Reply:
Sounds good to me!